src/main/java/pl/asie/foamfix/coremod/FoamFixTransformer.java

```diff
@@ -303,9 +303,6 @@ public static void init() {
             handlerCN.add(new EntityDataManagerPatch(), "net.minecraft.network.datasync.EntityDataManager");
         }
 
-        patchy.addTransformerId("disableTextureAnimations_v1");
```

Comment:
This breaks disabling texture animations when FoamFix's texture sprite patches are disabled...  as well as a fast global disable...

I like the idea, though - I might do an implementation myself at a point.

Reply:
You are right. I know this approach is far from perfect, but it is enough for what I need for my own experience, and I thought I might as well share it